Library/Formula/let-me-in.rb

```diff
@@ -0,0 +1,36 @@
+class LetMeIn < Formula
```

Comment:
You need:

```ruby
require "language/go"
```

At the top of this formula, one empty line above the `class` name.